src/Miki/resources/locales/eng.json

```diff
@@ -647,6 +647,7 @@
   "miki_command_usage_setnotifications": ">setnotifications [-type:(rewards only|all|none)] [-g:true]",
   "guildhouse_buy": "For your next upgrade. Your guild needs to pay {0} mekos! use `>guildupgrade house` to purchase it.",
   "error_user_null": "User not found!",
+  "error_bank_account_null": "Bank account doesn't exist!",
```

Comment:
the user doesn't know what a bank account is, this wouldn't make much sense to read